parachains/runtimes/assets/statemine/Cargo.toml

```diff
@@ -72,16 +73,22 @@ pallet-xcm-benchmarks = { git = "https://github.com/paritytech/polkadot", branch
 
 
 [dev-dependencies]
-hex-literal = "0.3.4"
 asset-test-utils = { path = "../test-utils"}
 
 [build-dependencies]
 substrate-wasm-builder = { git = "https://github.com/paritytech/substrate", branch = "master" }
 
 [features]
 default = [ "std" ]
+# When enabled the `state_version` is set to `1`.
+# This means that the chain will start using the new state format. The migration is lazy, so
+# it requires to write a storage value to use the new state format. To migrate all the other
+# storage values that aren't touched the state migration pallet is added as well.
+# This pallet will migrate the entire state, controlled through some account.
+#
+# This feature should be removed when the main-net will be migrated.
+state-trie-migration = ["pallet-state-trie-migration"]
```

Comment:
Why are we adding this feature? the next release/runttime should have this pallet, and once we are done with it, we can remove. 

I don't see why this requires a new feature. This might increase the chance of making a mistake and compiling different/wrong runtimes.

Reply:
I proposed to add the feature. Especially for changing the `state_version`. I don't want this in master until this in tested on Rococo/Westend. If that is done, we can remove the feature flag. Aka when we want to do this on main net. For the test nets I'm okay with the risk that someone forgets this.

Reply:
Okay, but let's make sure we have an issue to clear it later.

Reply:
What if we compile this runtime for a chain with this feature and later on make a mistake and not do it? What happens if we revert to state-version 0 again? @cheme 

Also, the feature should be named `state-version-1` or something similar.

Reply:
I am pretty sure we were running some test from v1 to v0 at some point, so I think we would migrate to 0.
Outside the migration question, all changes will be written in the old format. So then we probably would run another migration round to restore them to v1.
So sure will not be a good thing (can move a migrated chain back to hybrid state).